GVFS/GVFS.UnitTests/Maintenance/GitMaintenanceStepTests.cs

```diff
@@ -27,7 +27,6 @@ public void GitMaintenanceStepRunsGitAction()
         }
 
         [TestCase]
-        [Category(CategoryConstants.ExceptionExpected)]

```

Comment:
As discussed the naming here isn't great, but we want this message in.  ExceptionExpected means don't run in debug mode.